cypress/e2e/account/goToSettings.feature

```diff
@@ -1,16 +1,16 @@
 @release
-Feature: Account - To go to account settings page on default
+Feature: Account - To go to account settings page on dbc
```

Comment:
can you also update the file template.env.json in README ? the var name to 'DBC' (https://github.com/hpi-schul-cloud/e2e-system-tests/blob/main/env_variables/template.env.json)

Reply:
Its already done. See below

Reply:
Its already done. See below in this PR. For reference here is the file in my branch

 https://github.com/hpi-schul-cloud/e2e-system-tests/blob/BC-6251-ensure-consistent-instance-naming/env_variables/template.env.json